packages/cloudflare/src/cli/build/index.ts

```diff
@@ -276,3 +277,10 @@ export async function getLatestCompatDate(): Promise<string | undefined> {
     /* empty */
   }
 }
+
+function showErrorForOldNextjsVersions(options: buildHelper.BuildOptions) {
+  if (buildHelper.compareSemver(options.nextVersion, "14.0.0") < 0) {
+    logger.error("Next.js version unsupported, please upgrade to version 14 or greater.");
+    process.exit(1);
```

Comment:
Something I've picked up from @petebacondarwin, I usually prefer `throw`s  instead of hard `process.exit` calls so that we still have the possibility in the future if needed for whatever reason to catch it

Reply:
I'm fine with exiting.
We can change later if needed.